week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -73,6 +101,58 @@ function insertShoppingList(shoppingList) {
  */
 function insertBooks(books) {
   //Write your code in here
+  let content = document.querySelector("#content");
+  let list = document.createElement('ul');
+
+  for(let i = 0; i < books.length; i++){
+    let infoEls = document.createElement('li');
+    infoEls.innerHTML = books[i].title + " -" + books[i].author;
+    
+    content.appendChild(infoEls);
+    
+  }
+  content.appendChild(list);
+
+ let imgArr = ["https://miro.medium.com/max/1000/1*Qo27inBKBKY4Q4Pgk5KkbQ.png",
+                "https://images-na.ssl-images-amazon.com/images/I/51aKPFuUy1L._SX324_BO1,204,203,200_.jpg",
+                "https://imagery.pragprog.com/products/595/tpp20.jpg?1557173963"];
+
+ for(let j = 0; j <books.length; j++){
+ let paraEls = document.createElement('p');
+ 
+ //paraEls.innerHTML = books[j].title + " -" + books[j].author;
+
+ let imgEls = document.createElement('img');
+
+ let linkEls = document.createElement('a');
+
+ paraEls.innerHTML = books[j].title + " -" + books[j].author + linkEls;
```

Comment:
I'd also recommend trying out a template literal here with this one 😄 
The link is [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)